docs/quickstart.rst

```diff
@@ -28,59 +31,128 @@ Lets get started from the feature engineering in `Spark`_.
         SparkSession
         .builder
         .appName("rikai-quickstart")
-        .config('spark.jars.packages', 'ai.eto:rikai:0.0.5')
-        .config("spark.driver.extraJavaOptions", "-Dcom.amazonaws.services.s3.enableV4=true")
-        .config('spark.jars.packages', 'org.apache.hadoop:hadoop-aws:2.7.4')
-        .master("local[*]")
         .getOrCreate()
     )
 
     df = spark.createDataFrame(
         [
             {
                 "id": 1,
-                "mat": DenseMatrix(2, 2, range(4)),
                 "image": Image("s3://foo/bar/1.png"),
                 "annotations": [
                     Row(
                         text="cat",
+                        label=2,
                         mask=np.random(size=(256, 256)),
-                        bbox=Box2d(x=1.0, y=2.0, width=3.0, height=4.0)
+                        bbox=Box2d(xmin=1.0, ymin=2.0, xmax=3.0, ymax=4.0)
                     )
                 ]
             },
         ]
     )
 
-    df.write.format("rikai").save("dataset/out")
+    df.write.format("rikai").save("my_dataset")
 
-We can then inspect the dataset in a `Jupyter Notebook`_.
+The magic here is that Rikai maintains commonly used :doc:`Semantic Types <types>`
+and takes care of ``SerDe`` (short for **Ser**-ialization and **De**-serialization)
```

Comment:
i think if potential users are here they should understand what serde stands for